src/town_gui.cpp

```diff
@@ -1511,6 +1513,9 @@ class HousePickerCallbacks : public PickerCallbacks {
 	int GetSelectedType() const override { return sel_type; }
 	void SetSelectedType(int id) const override { sel_type = id; }
 
+	bool GetHousePickerProtected() { return house_protected; }
+	void SetHousePickerProtected(bool make_protected) const { house_protected = make_protected; }
```

Comment:
```suggestion
	bool IsHousePickerProtected() { return this->house_protected; }
	void SetHousePickerProtected(bool make_protected) const { this->house_protected = make_protected; }
```
Definitely missing `this->`. Maybe `Is...` is better naming wise than `Get...`, as that kinda tells you it returns a boolean value.

Reply:
`house_protected` is static so it shouldn't have `this->`